benchmarks/src/main/scala/zio/GenBenchmarks.scala

```diff
@@ -0,0 +1,81 @@
+package zio
+
+import java.util.concurrent.TimeUnit
+
+import hedgehog.core.GenT
+import org.openjdk.jmh.annotations._
+import org.scalacheck
+import zio.random.Random
+import zio.test.Gen
+
+@State(Scope.Thread)
+@BenchmarkMode(Array(Mode.AverageTime))
+@OutputTimeUnit(TimeUnit.NANOSECONDS)
+class GenBenchmarks {
+
+  @Param(Array("1000"))
+  var size: Int = _
+
+  @Benchmark
+  def zioDouble: Gen[Random, Double] =
+    Gen.double(0, size)
+
+  @Benchmark
+  def zioIntListOfSizeN: Gen[Random, List[Int]] =
+    Gen.listOfN(size)(Gen.anyInt)
+
+  @Benchmark
+  def zioStringOfSizeN: Gen[Random, String] =
+    Gen.stringN(size)(Gen.anyChar)
+
+  @Benchmark
+  def scalaRandomDouble: Double =
```

Comment:
It's not useful to include these in the benchmark, they aren't `Gen`-like.

Reply:
#1630 
`generate common random data types with ZIO Test versus other property based testing frameworks and natively using scala.util.Random.`

do you mean we don't want to use scala.util.Random here at all anymore or am I just using it incorrectly in the benchmark?